docs/README.md

```diff
@@ -56,7 +56,37 @@ Each chapter must link a file: internal links to anchors within files do not
 work. There is an [open issue](https://github.com/rust-lang/mdBook/issues/167)
 to fix this behavior.
 
+### TLA+ syntax highlighting
+
+We provide a custom TLA+ syntax highlighting plugin for [highlight.js][] in `highlightjs-tlaplus`.
+
+#### Building
+
+To build, check out the main highlight.js into a separate directory.
+Symlink the TLA+ language definition into the highlight.js `extra` directory:
```

Comment:
I think it could help to clarify for readers that they don't need to worry about building this unless they need to update it. So perhaps add a line like:
```suggestion

The plugin is checked in with the source code, and you generally shouldn't need to rebuild it unless you need to make a change or update the plugin.

To build, check out the main highlight.js into a separate directory.
Symlink the TLA+ language definition into the highlight.js `extra` directory:
```
 
Does this make sense?

Reply:
That's a very thoughtful addition, thank you!